certified-connectors/ITautomate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,272 @@
+    "contact": {
+      "name": "ITautomate Support",
+      "url": "https://www.itautomate.io/contact-us/",
+      "email": "support@itautomate.io"
+    }
+  },
+  "host": "connectorapi.itautomate.io",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/RunCommand": {
+      "post": {
+        "tags": [
+          "DynamicCommand"
+        ],
+        "summary": "Issue PowerShell Command",
+        "description": "Issue a saved ITautomate PowerShell command to remote device(s)",
+        "operationId": "RunCommand",
+        "consumes": [
+          "application/json",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "description": "Issue a saved ITautomate PowerShell command to remote device(s)",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "format": "int32",
+            "x-ms-summary": "id",
+            "x-ms-dynamic-values": {
+              "operationId": "ListAllCommands",
+              "parameters": {},
+              "value-collection": "Commands",
+              "value-path": "Id",
+              "value-title": "Name"
+            }
+          },
+          {
+            "name": "commandInput",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommandInputParameter"
+            },
+            "x-ms-summary": "CommandInputParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommandRunResultModel_X"
+            }
+          },
+          "default": {
+            "description": "OK",
+            "schema": {}
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/OutputFieldsSchema": {
+      "get": {
+        "tags": [
+          "DynamicCommand"
+        ],
+        "summary": "OutputFieldsSchema",
+        "operationId": "OutputFieldsSchema",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          },
+          "400": {
+            "description": "Invalid commandId"
+          },
+          "default": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@ITautomate please remove an empty schema here as well. You can also change the description value here for the default response if you want.